PendingReleaseNotes

```diff
@@ -318,3 +318,8 @@
   or manually without ceph-volume and LVM.  Going forward, the OSD
   will limit itself to only 'hdd' and 'ssd' (or whatever device class the user
   manually specifies).
+
+* The configuration value ``osd_calc_pg_upmaps_max_stddev`` has been removed.
+  Instead use the mgr balancer config ``upmap_max_deviation`` which now is an
```

Comment:
Let's also mention the current default value and the scenarios when users might want to set this to a higher value(e.g. unequal hardware sizes https://tracker.ceph.com/issues/42971#note-4)